tests/testsuite/build_script.rs

```diff
@@ -355,16 +355,7 @@ fn custom_build_env_var_rustc_linker_bad_host_target() {
                 target
             ),
         )
-        .file(
-            "build.rs",
-            r#"
-            use std::env;
-
-            fn main() {
-                assert!(env::var("RUSTC_LINKER").unwrap().ends_with("/path/to/linker"));
-            }
-            "#,
-        )
+        .file("build.rs", "fn main() {}")
```

Comment:
To confirm, this is being deleted because the build script is never actually running, right?

If so that makes me wonder where this came from and what it was originally intended for, but that can be left to another time.

Reply:
Yes, that's right. I suspect it was a case of copy-pasting test cases to check very related things, and not realizing the nature of the modified test was such that the content of the build script was irrelevant.

Reply:
> If so that makes me wonder where this came from and what it was originally intended for, but that can be left to another time.

> I suspect it was a case of copy-pasting test cases to check very related things, and not realizing the nature of the modified test was such that the content of the build script was irrelevant.

Yes, they were copy pasted, I had left some non-required parts in the tests to make them more similar to other tests(mostly to make it easier for me to reason about what build variations have test coverage by looking at just the differences between similar tests), it should be safe to clean those parts up.